migration/iD10/postgresql/202208301110_IDEMPIERE-5400.sql

```diff
@@ -0,0 +1,7 @@
+-- IDEMPIERE-5400
+SELECT register_migration_script('202208301110_PlaceholderForTicket.sql') FROM dual;
```

Comment:
you forgot to change the name here to match with the sql script name.  same goes for the oracle script above